uiowa/uiowa
# Generate uids_base sourcemaps based on environment and fix gulp watch.

Resolves #1289 

This PR:
- removes browsersync because it will not work on Vagrant mounted drives.
- Fixes the uids_base Gulp watch task by using polling. This is very slow however, again because of Vagrant mounted drives.
- Removes requirement on unused susy dependency.

### Testing
- Run `blt frontend` and verify that sourcemaps are not generated.
  - This just runs `yarn workspace uids_base run build` which is another option.
- Run `yarn workspace uids_base run watch` and verify that sourcemaps are generated after changing a file.

Sounds like DrupalVM/Vagrant users run frontend tasks on their host which I don't get because NPM packages like node-sass have platform-dependent installations. Maybe we can work that out in the future to speed up processes that watch for filesystem changes, e.g Gulp watch.
